mteb/tasks/Audio/AudioClassification/eng/SpeechCommands.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import annotations
+
+from mteb.abstasks.Audio.AbsTaskAudioClassification import (
+    AbsTaskAudioClassification,
+)
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class SpeechCommandsClassification(AbsTaskAudioClassification):
+    metadata = TaskMetadata(
+        name="SpeechCommands",
+        description="A set of one-second .wav audio files, each containing a single spoken English word or background noise.",
+        reference="https://arxiv.org/abs/1804.03209",
+        dataset={
+            "path": "AdnanElAssadi/speech_commands_small",
+            "revision": "a59564b91bf0cfcf587e11c2603fe42bae21e5f0",  # Using downsampled version of v0.02
```

Comment:
```suggestion
            "revision": "a59564b91bf0cfcf587e11c2603fe42bae21e5f0",
```
How it was downsamples should be explained in the description. E.g. 

"To keep evaluations fast, this dataset was downsampled to N test samples."